rascal-vscode-extension/src/test/vscode-suite/utils.ts

```diff
@@ -464,42 +470,40 @@ async function assureDebugLevelLoggingIsEnabled() {
     await prompt.confirm();
 }
 
-export function printRascalOutputOnFailure(channel: 'Language Parametric Rascal' | 'Rascal MPL') {
-
+export async function printRascalOutputOnFailure(context: Mocha.Context, ide: IDEOperations, channel: 'Language Parametric Rascal' | 'Rascal MPL') {
```

Comment:
can this function now become like this again? to do an after each?

Reply:
Sure, I reverted it. That also adresses your other comments regarding duplication. It means we lose the screenshot at failure time, but that was not that useful anyway.